streampark-console/streampark-console-service/src/main/resources/mapper/core/ApplicationMapper.xml

```diff
@@ -98,6 +99,14 @@
         limit 1
     </select>
 
+    <select id="existsRunningJobByClusterId" resultType="java.lang.Boolean" parameterType="java.lang.Long">
+        select count(1)
+        from t_flink_app
+        where flink_cluster_id = #{clusterId}
```

Comment:
I didn't understand, what is the difference between `cluster_id` and `flink_cluster_id`

Reply:
> I didn't understand, what is the difference between `cluster_id` and `flink_cluster_id`

cluster_id: Such as the applicationId of the yarn-session and the k8s cluster Id
flink_cluster_id: Clusters managed by streampark, from t_flink_cluster id field